docs/put-response-codes.md

```diff
@@ -1,40 +1,32 @@
-# CreateOperationAsyncResponseValidation
+# PutResponseCodes
 
 ## Category
 
-RPaaS Error
+ARM Error
 
 ## Applies to
 
-ARM OpenAPI(swagger) specs
+ARM OpenAPI (swagger) specs
 
 ## Related ARM Guideline Code
 
 - RPC-Async-V1-01
 
 ## Output Message
 
-202 is not a supported response code for PUT async response
+Synchronous and Long-running PUT operations must have responses with 200, 201 and default return codes. They also must not have other response codes.
 
 ## Description
 
-An async PUT operation response must include status code 201 for creation. It must also support status code 200, for replacement. The operation must also add "x-ms-long-running-operation" to mark that it is a long running operation.
+Synchronous and Long-running PUT operations must have responses with 200, 201 and default return codes. They also must not have other response codes.
 
-## CreatedAt
-
-August 10, 2020
-
-## LastModifiedAt
-
-August 10, 2020
-
-## Why this rule is important
+## How to fix the violation
 
-RPaaS only supports 201 for async PUT operations. This is enforced at runtime via swagger validation.
+For synchronous and long-running (LRO) PUT operations, specify responses with 200 & 201 return codes.
 
-## How to fix the violation
+### Example
 
-Add the following for async PUT operations.
+Add the following to PUT operations.
 
 The following would be valid:
```

Comment:
Lets provide 2 examples. One for sync and one for async with the only difference being that the sync example will not have "x-ms-long-running-operation": true